edx_proctoring/templates/proctoring/seq_proctored_exam_rejected.html

```diff
@@ -0,0 +1,55 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session was reviewed and did not pass requirements
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b class="failure"> Failed </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      Your session was reviewed by one of our proctors and some activities noted violate the
```

Comment:
"Your proctoring session data was reviewed and there were either issues with your identity verification or evidence of suspicious activity during the exam."